### ISSUE TITLE
streamsilk new resolver

### ISSUE THREAD

**Issue by Ghb3245** [Gujal00/ResolveURL#884]
`https://streamsilk.com/p/66debc984b1c4`

**Comment by Ghb3245** [comment]
sorry is it playable in kodi because i always get this error playing fine on browser :
`2024-09-13 21:08:24.003 T:9772  warning <general>: ResolveURL: Stream UrlOpen Failed: Url: https://ss12.silkanu.cc/data/9/66e3eb81c1b0b/master.m3u8|User-Agent=Mozilla%2F5.0+%28Windows+NT+10.0%3B+Win64%3B+x64%29+AppleWebKit%2F537.36+%28KHTML%2C+like+Gecko%29+Chrome%2F118.0.0.0+Safari%2F537.36&Referer=https%3A%2F%2Fstreamsilk.com%2F&Origin=https%3A%2F%2Fstreamsilk.com HTTP Code: 404 Msg: `

**Comment by Gujal00** [comment]
Yes it has been purposefully broken now by the site for resolveurl. Have to do a new resolver and probably would be broken by the site again as it looks like they are watching this git
